styles/styles.css

```diff
@@ -358,6 +381,243 @@ h2.advantages__heading:after {
   stroke-width: 2;
 }
 
+/* DESTINATIONS */
+.destinations {
+  max-width: 1440px;
+  max-height: 958px;
+  height: inherit;
+  padding-top: 120px;
+  padding-bottom: 208px;
+  margin: 0 auto;
+}
+
+.destinations__tabs {
+  width: inherit;
+  max-width: calc(192px + 192px + 238px);
```

Comment:
in this case you can just use max-width:622px. But you can use whatever you want, even if it's looking weird)

Reply:
> in this case you can just use max-width:622px

Fixed 👉 https://github.com/pavel-4/if-html/pull/12/commits/2c1cf0dc0200b9ce9ffb87cffed1a582b9dd3cc5 cc @almuly